packages/tabs/README.md

```diff
@@ -108,6 +61,133 @@ _Any other properties supplied will be spread on the root element._
 | `children`        | `node`                                             | Content that appears inside the `<Tab />` component                                                                       |         |
 | ...               | native attributes of component passed to `as` prop | Any other props will be spread on the root element                                                                        |         |
 
+# Test Harnesses
+
+## getTestUtils()
+
+`getTestUtils()` is a util that allows consumers to reliably interact with `LG Tabs` in a product test suite. If the `Tabs` component cannot be found, an error will be thrown.
+
+### Usage
+
+```tsx
+import { Tabs, getTestUtils } from '@leafygreen-ui/tabs';
+
+const utils = getTestUtils(lgId?: string); // lgId refers to the custom `data-lgid` attribute passed to `Tabs`. It defaults to 'lg-tabs' if left empty.
+```
+
+#### Single `Tabs` component
+
+```tsx
+import { render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { Tabs, getTestUtils } from '@leafygreen-ui/tabs';
+
+...
+
+test('tabs', () => {
+  render(
+    <Tabs aria-label="Label">
+      <Tab name="First" default>
+        Content 1
+      </Tab>
+      <Tab name="Second">
+        Content 2
+      </Tab>
+      <Tab name="Third">
+        Content 3
+      </Tab>
+    </Tabs>
+  );
+
+  const { getAllTabsInTabList, getTabUtilsByName, getSelectedPanel } = getTestUtils();
```

Comment:
I'm curious, why is it called TabUtil?

Reply:
I was trying to follow a similar pattern to `FormUtils`: https://github.com/mongodb/leafygreen-ui/blob/c3906f78b64758dd5a882f75ef9fa25c8956702b/tools/test-harnesses/src/types/form/formUtils.types.ts#L1